docs/content/getting-started/production-best-practices.mdx

```diff
@@ -21,17 +21,53 @@ The following list outlines some guidelines and best practices for running OpenF
 - Set the log.format to "json"
 - Set the log.level to "info"
 - [Disable the Playground](./setup-openfga/docker-setup.mdx#playground).
+- [Set Concurrency Limits](#concurrency-limits)
 
 ## Database Recommendations
 
 To ensure good performance for OpenFGA, it is recommended that the [database](./setup-openfga/docker-setup.mdx#configuring-data-storage) be:
 - Used exclusively for OpenFGA and not shared with other applications. This allows scaling the database independently and avoiding contention with your database.
 - Bootstrapped and managed with the `openfga migrate` command. This will ensure the appropriate database indexes are created.
 
-In addition, it's strongly recommended to fine-tune your server settings:
+It's strongly recommended to fine-tune your server database connection settings to avoid having to re-establish database connections frequently. Establishing database connections is slow and will negatively impact performance, and so here are some guidlines for managing database connection settings:
 
 - The server setting `OPENFGA_DATASTORE_MAX_OPEN_CONNS` should be set to be equal to your database's max connections. For example, in Postgres, you can see this value via running the SQL query `SHOW max_connections;`. If you are running multiple instances of the OpenFGA server, you should divide this setting equally among the instances. For example, if your database's `max_connections` is 100, and you have 2 OpenFGA instances, `OPENFGA_DATASTORE_MAX_OPEN_CONNS` should be set to 50 for each instance.
 
+- The `OPENFGA_DATASTORE_MAX_IDLE_CONNS` should be set to a value no greater than the maximum open connections (see the bullet point above), but it should be set sufficiently high enough to avoid having to recreate connections on each request.
+
+  If you see a lot of database connections being closed and subsequently reopened, then you should consider increasing the maximum number of idle connections.
```

Comment:
Re: "If you see a lot of database connections being closed and subsequently reopened"

We should add instructions on how to see this. But today, how can they do that? Wouldn't we have to expose DB stats from Openfga? https://pkg.go.dev/database/sql#DBStats

Reply:
We currently don't expose the DBStats. The user can however monitor this from their DB stats as part of operations.

I have a little bit of hesitation in exposing this as this is specific to SQL base stats. Other database connector implementation will not have this available (or have the same struct).

Reply:
> The user can however monitor this from their DB stats as part of operations.

Yeah that's what I thought, but i think we should mention that here.